VTUL/iawa
# LIBTD-1653: In public collection view, move up subcollection list and…

… move down collection details.

**Rearrange the public Collection view so to reduce having to scroll down to view subcollections.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1653) (:star:)

# What does this Pull Request do? (:star:)
To reduce having to scroll down to view subcollections, customize the public collection view page: list subcollections right below parent collections and move collection details from the right of the parent collections down to below subcollections. 

# What's the changes? (:star:)

* Move the div for showing collection details to below subcollection div in app/views/hyrax/collections/show.html.erb.
* From 
<img width="1042" alt="libtd-1653_before" src="https://user-images.githubusercontent.com/8228221/49103911-a08e9280-f24a-11e8-9271-6ed4a06fd4bf.png">
to
<img width="1351" alt="libtd-1653_after" src="https://user-images.githubusercontent.com/8228221/49103923-a6847380-f24a-11e8-81a1-240a895e918d.png">

# How should this be tested?

* Create a collection (_the_collection_) with some description metadata.
* Create several parent collections, sub collections and works for _the_collection_ created above.
* Either from Dashboard --> Collections --> _the_collection_ --> Public view of Collection or from Home --> Explore Collections --> _the_collection_ ,  to see the rearranged view.

# Additional Notes:

* Branch LIBTD-1653

# Interested parties
Tag (@shabububu ) interested parties

(:star:) Required fields
